pages/portfolio.html

```diff
@@ -5,8 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
```

Comment:
BTW, the HTML validator had nothing to flag in both this and contact.html! Good work!